lms/envs/aws.py

```diff
@@ -729,6 +729,7 @@
 #### JWT configuration ####
 JWT_ISSUER = ENV_TOKENS.get('JWT_ISSUER', JWT_ISSUER)
 JWT_EXPIRATION = ENV_TOKENS.get('JWT_EXPIRATION', JWT_EXPIRATION)
+JWT_AUTH.update(ENV_TOKENS.get('JWT_AUTH', JWT_AUTH))
```

Comment:
If `ENV_TOKENS` has no JWT_AUTH key, you will update `JWT_AUTH` with itself. Use an empty dict as the default.